models/calendar.yml

```diff
@@ -37,7 +37,9 @@
         title: Introductions
         emoji: 👋
 
-    - 7:45 PM: Speaker TBA
+    - 7:45 PM:
```

Comment:
Looking good! Would you mind adding in a few more bits of information? This is the usual format:

```yml
    - 7:45 PM:
        type: Talk
        emoji: 📞
        title: 'Your Title Here'
        description: 'Your description here.'
        person:
          name: Foo Bar
          twitter: foobar
```